tests/affine-cloud-copilot/e2e/settings/embedding.spec.ts

```
@@ -134,9 +136,17 @@ test.describe('AISettings/Embedding', () => {
     loggedInPage: page,
     utils,
   }) => {
+    {
+      // use identical workspace
+      await utils.settings.closeSettingsPanel(page);
+      await createLocalWorkspace({ name: 'test' }, page, false, 'affine-cloud');
+    }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)